src/main/java/educationalproject/programmingstuff/servicies/dto/OrderItemResponseDto.java

```diff
@@ -0,0 +1,16 @@
+package educationalproject.programmingstuff.servicies.dto;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class OrderItemResponseDto {
+
+    long orderID;
```

Comment:
final ... orderId

Reply:
and btw, do we need orderId in each OrderItemResponseDto, because we already have this information in the parent dto?

Reply:
Done